cookbooks/new_ping.md

```diff
@@ -0,0 +1,45 @@
+Got some new data you want to send to us? How in the world do you send a new ping? Follow this guide to find out.
+
+Write Your Questions
+--------------------
+Do not try and implement new pings unless you know specifically what questions you're trying to answer. General questions about "How do users use our product?" won't cut it - these need to be specific, concrete asks that can be translated to data points. This will also make it easier down the line as you start data review.
+
+Create a Schema
+---------------
+Use JSON Schema to start with. See the examples schemas in the [Mozilla Pipeline Schemas repo](https://github.com/mozilla-services/mozilla-pipeline-schemas/).
```

Comment:
For JSON and Parquet schemas it will be good to list all templates somewhere which could be used in shema (by templates I mean anything starting with "@" and then unfolds to custom lines of schema)